.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,48 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the Tarantool build artifact
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Clone the crud module
+        uses: actions/checkout@v3
+        with:
+          repository: slavakirichenko/crud
+          ref: slavakirichenko/add-reusable-workflow-testing
+
+      - name: Download the Tarantool build artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.artifact_name }}
+
+      - name: Install Tarantool
+        run: |
+          sudo dpkg -i tarantool_*.deb tarantool-common_*.deb tarantool-dev_*.deb
+          tarantool --version
+
+      - name: Install requirements
+        run: ./deps.sh
+
+      - name: Install metrics
+        run: tarantoolctl rocks install metrics 0.12.0
+
+      - run: echo $PWD/.rocks/bin >> $GITHUB_PATH
```

Comment:
?

Reply:
There's no such step in the original workflow https://github.com/tarantool/crud/blob/master/.github/workflows/check_on_push.yaml

Reply:
Fixed.